### PR TITLE
README.md updated based on feedback provided in #126

### PR DESCRIPTION
**Pull request by jufemaiz**
Updated the README.md based on https://github.com/GoogleContainerTools/container-structure-test/issues/126 to reduce confusion.

### CODE REVIEW

**Comment by jufemaiz** [comment]
cla/google :taco: I've tried but it won't let me do it for our GSuite account.

Doing it for personal email.

File: README.md [hunk 1]
```diff
@@ -240,12 +262,26 @@ steps:
...
 # Test the image.
 - name: 'gcr.io/gcp-runtimes/container-structure-test'
-  args: ['-image', 'gcr.io/$PROJECT_ID/image', 'test_config.yaml']
+  args: ['test', '--image', 'gcr.io/$PROJECT_ID/image', '--config', 'test_config.yaml']
 
 # Push the image.
 images: ['gcr.io/$PROJECT_ID/image']
 ```
 
+## Running File Tests via Docker Generically
```
[the remark this answers is not in the document]
Comment (jufemaiz) [reply README.md]: The reason i put it in is that we need to do this to run on our CICD and it took me a while to discover.

Perhaps we should have a note that it is not recommended?

[the remark this answers is not in the document]
Comment (nkubala) [reply README.md]: Could you explain your use case a little bit more? I think there should be a way to run in most environments without doing this.

[the remark this answers is not in the document]
Comment (zemanlx) [reply README.md]: I am running it on Travis Linux machines, but my policy is that everybody in my team should be able to run all tests locally and as far as half of our developers are OSX users and there is only Linux binary I need something that everybody can run, therefore I have to run it in Docker.

[the remark this answers is not in the document]
Comment (jufemaiz) [reply README.md]: See @zemanlx's comment.

[the remark this answers is not in the document]
Comment (nkubala) [reply README.md]: Yeah that's fair. I actually just updated the README a few days ago to provide instructions on building a binary for OSX. Unfortunately Bazel's cross compilation is funky which is why we don't publish an OSX binary currently, but I'm working on trying to get that fixed. I might just resort to building and uploading them myself. But I'd rather do that than encourage people to run docker in docker :)

[the remark this answers is not in the document]
Comment (nkubala) [reply README.md]: @zemanlx @jufemaiz just an FYI, as of release v1.2.0 we now publish an OSX binary. Please give it a shot and let me know if you have any issues

**Comment by dlorenc** [comment]
I think this is obsolete now that we publish OSX binaries.

**Pull request closed without merging**